pkg/kates/client.go

```diff
@@ -357,26 +358,71 @@ func isExpiredError(err error) bool {
 }
 
 type lw struct {
```

Comment:
I know this isn't a change in this PR, but I just realized how much happier I'd've been to have this be `type listWatcher struct { ... }`.